starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -372,39 +370,12 @@ sub set_maneuver {
                       sprintf(
                         "Uplink quaternion norm value $norm is too far from 1.0\n");
                 }
-                my @c_quat_norm = (
-                    $c->{Q1} / $norm,
-                    $c->{Q2} / $norm,
-                    $c->{Q3} / $norm,
-                    $q4_obc / $norm
-                );
-
-               # Get quat from MANEUVER summary file.  This is correct to high precision
-                my $q_man = Quat->new($m->{ra}, $m->{dec}, $m->{roll});
-                my $q_obc = Quat->new(@c_quat_norm);
-                my @q_man = @{ $q_man->{q} };
-                my $q_diff = $q_man->divide($q_obc);
 
-                if (   abs($q_diff->{ra0} * 3600) > 1.0
-                    || abs($q_diff->{dec} * 3600) > 1.0
-                    || abs($q_diff->{roll0} * 3600) > 10.0)
-                {
-                    push @{ $self->{warn} },
-                      sprintf(
-"Target uplink precision problem for MP_TARGQUAT at $c->{date}\n"
```

Comment:
This PR removes the uplink precision check as I'm not sure what to check if we don't read the maneuver summary for comparison.

Reply:
I'm fine with removing the uplink precision check.